added an example of source element using height & width attributes (#21819)
mdn/content: files/en-us/mozilla/firefox/experimental_features/index.md

  </tbody>
</table>

### Height & Width attributes for `<source>`

HTML attributes for [height](/en-US/docs/Web/HTML/Element/source#attr-height) & [width](/en-US/docs/Web/HTML/Element/source#attr-width) ({{bug(1694741)}}).

> This works when the {{HTMLElement("source")}} when it is a child element of {{HTMLElement("picture")}} and helps eliminate layout shift.

Reviewer: ```suggestion
These attributes can be used in {{HTMLElement("source")}} elements when they are children of {{HTMLElement("picture")}} elements and help to eliminate layout shift.
```
